### ISSUE TITLE
New color scheme (theme) for the blog

### ISSUE THREAD

**Issue by huytd** [huytd/azeroth-js#8]
We have the `css/theme.css` that define the current style of the blog, with red and white tone.

For some people, this is kind of boring.

So we can create a new CSS file to define a new color scheme (for headers, text, link, logo, code,...)

This work only required some basic CSS skills.

**Comment by umer936** [comment]
I can do this. How fancy were you thinking?

**Comment by goelrohan6** [comment]
Kindly have a look at #10

**Comment by huytd** [comment]
@goelrohan6 has submitted a Pull Request so I close this one.

But more theme can be able to submitted. Just refer to this issue :D

**Comment by huytd** [comment]
Actually, just keep this issue here so everybody can continue add new PRs

**Comment by RaiVaibhav** [comment]
hope you like BLACK and White theme

**Comment by huytd** [comment]
I'll merge when September ends 😂

**Comment by RaiVaibhav** [comment]
@huytd  Sir i yesterday i have made some commit and  out of which 3-4 gets merge so this will not count in  hacktoberfest???

**Comment by huytd** [comment]
Yeah, I've read somewhere said that hacktoberfest only count for the merge happen during october :D

**Comment by Swayamsvk** [comment]
Can I work on this?

**Comment by huytd** [comment]
@Swayamsvk yes, please feel free to open a new ticket and link it here, there's no limit of the themes.

**Comment by mxwllndrsn** [comment]
Submitted Pull Request #30 for this issue.
